### DISCUSSION TITLE
connection refused when accessing mediamtx in docker.

### DISCUSSION THREAD

**Discussion by Lunewcome** [bluenviron/mediamtx#3589] [Questions]
### Question

1. I downloaded a release version from :"https://github.com/bluenviron/mediamtx/releases/download/v1.8.4/mediamtx_v1.8.4_linux_amd64.tar.gz".
2.  My docker images is centos7.
3. docker start cmd:
 docker run -it --network=host --name mmtx  -vD:\docker-shared:/home/shared as-env:1.0 /bin/bash
4. and mediamtx can be started normally:
[root@docker-desktop mmtx]# ./mediamtx
2024/07/31 14:51:39 INF MediaMTX v1.8.4
2024/07/31 14:51:39 INF configuration loaded from /root/mmtx/mediamtx.yml
2024/07/31 14:51:39 INF [path ztxy1] [RTSP source] started
2024/07/31 14:51:39 INF [RTSP] listener opened on :8554 (TCP), :8000 (UDP/RTP), :8001 (UDP/RTCP)
2024/07/31 14:51:39 INF [RTMP] listener opened on :1935
2024/07/31 14:51:39 INF [HLS] listener opened on :8888
2024/07/31 14:51:39 INF [WebRTC] listener opened on :8889 (HTTP), :8189 (ICE/UDP)
2024/07/31 14:51:39 INF [SRT] listener opened on :8890 (UDP)
2024/07/31 14:51:39 INF [path ztxy1] [RTSP source] ready: 2 tracks (H265, MPEG-4 Audio)
2024/07/31 14:51:42 WAR [path ztxy1] [RTSP source] no UDP packets received, switching to TCP

5. But when I try to visity "http://127.0.0.1:8889/ztxy1" in browser, it always says "connection refused".

I have tried run the server in windows locally, and it works fine.

**Comment by Lunewcome** [comment]
I try like this :
docker run --rm -it \
-e MTX_PROTOCOLS=tcp \
-e MTX_WEBRTCADDITIONALHOSTS=192.168.x.x \
-p 8554:8554 \
-p 1935:1935 \
-p 8888:8888 \
-p 8889:8889 \
-p 8890:8890/udp \
-p 8189:8189/udp \
bluenviron/mediamtx

and it works!

**Comment by aler9** [comment]
answered here: https://github.com/bluenviron/mediamtx/discussions/3664#discussioncomment-10557303
